yast/src/include/smt/wizard.rb

```diff
@@ -260,7 +260,7 @@ def MainSequence(sequence_type)
 
       if sequence_type == "config"
         # TRANSLATORS: dialog caption
-        caption = _("NU Configuration")
+        caption = _("Update Server Configuration")
```

Comment:
No. Registration server is scc.suse.com, while update server is updates.suse.com, or?